### ISSUE TITLE
use backspace to delete checked item error in TreeSelect with labelInValue

### ISSUE THREAD

**Issue by daskyrk** [ant-design/ant-design#8423]
### Version
2.13.11

### Environment
antd: 2.13.10 
chrome: 62

### Reproduction link
[https://codepen.io/daskyrk/pen/LOaYKq?editors=0011](https://codepen.io/daskyrk/pen/LOaYKq?editors=0011)

### Steps to reproduce
in multiple mode, set `labelInValue=true`, check any node, then use backspace to delete it

### What is expected?
no error

### What is actually happening?
throw error

---
### reason:
in function `getDeselectedValue`:
```javascript
    var unCheckPos = void 0;
    checkedTreeNodes.forEach(function (itemObj) {
      if (itemObj.node.props.value === selectedValue) {
        unCheckPos = itemObj.pos;
      }
    });
```
selectedValue is a object {label, value}, but props.value is not

**Comment by danedavid** [comment]
Good catch. I would also add "Delete the node" in the expected behavior.

**Comment by zombieJ** [comment]
Fixed in `rc-tree-select@2.0.10`
